### ISSUE TITLE
nach update auf 2.5: error: heatingcontrol.0 (22372) exception in CheckExternalStates [TypeError: Cannot read property 'val' of null]

### ISSUE THREAD

**Issue by looxer01** [rg-engineering/ioBroker.heatingcontrol#255]
Hi 
Ich bekommen den untenstehenden Fehler immer, wenn Heatingcontrol neu gestartet wird.
Protokollausschnitt untenstehend.
vG Looxer

`
<html>
<body>

021-08-20 09:29:27.489 - info: heatingcontrol.0 (22372) check all active sensors
--
2021-08-20 09:29:27.491 - debug: heatingcontrol.0 (22372) Sensor state false ?= false
2021-08-20 09:29:27.492 - debug: heatingcontrol.0 (22372) boolean: one window closed in Wohnzimmer hm-rpc.0.MEQ1727200.1.STATE
2021-08-20 09:29:27.492 - debug: heatingcontrol.0 (22372) result for CheckSensors4Room in Wohnzimmer is false since 1629441652028 = never
2021-08-20 09:29:27.493 - debug: heatingcontrol.0 (22372) Sensor state false ?= false
2021-08-20 09:29:27.493 - debug: heatingcontrol.0 (22372) boolean: one window closed in Schlafzimmer hm-rpc.0.MEQ0183268.1.STATE
2021-08-20 09:29:27.494 - debug: heatingcontrol.0 (22372) result for CheckSensors4Room in Schlafzimmer is false since 1629443239983 = never
2021-08-20 09:29:27.495 - debug: heatingcontrol.0 (22372) Sensor state 0 ?= false
2021-08-20 09:29:27.496 - debug: heatingcontrol.0 (22372) boolean: one window closed in Elternbad hm-rpc.1.0000D709954526.1.STATE
2021-08-20 09:29:27.496 - debug: heatingcontrol.0 (22372) result for CheckSensors4Room in Elternbad is false since 1629443252708 = never
2021-08-20 09:29:27.497 - debug: heatingcontrol.0 (22372) result for CheckSensors4Room in Hobbyraum is false since 0 = never
2021-08-20 09:29:27.497 - debug: heatingcontrol.0 (22372) result for CheckSensors4Room in Gaeste WC is false since 0 = never
2021-08-20 09:29:27.497 - debug: heatingcontrol.0 (22372) result for CheckSensors4Room in Jana is false since 0 = never
2021-08-20 09:29:27.497 - debug: heatingcontrol.0 (22372) result for CheckSensors4Room in Teresa is false since 0 = never
2021-08-20 09:29:27.497 - debug: heatingcontrol.0 (22372) result for CheckSensors4Room in Flur is false since 0 = never
2021-08-20 09:29:27.497 - debug: heatingcontrol.0 (22372) result for CheckSensors4Room in Kueche is false since 0 = never
2021-08-20 09:29:27.498 - info: heatingcontrol.0 (22372) checking external states
2021-08-20 09:29:27.500 - debug: heatingcontrol.0 (22372) HeatingPeriodActive false
2021-08-20 09:29:27.545 - debug: heatingcontrol.0 (22372) Present true
2021-08-20 09:29:27.546 - debug: heatingcontrol.0 (22372) VacationAbsent false
2021-08-20 09:29:27.547 - error: heatingcontrol.0 (22372) exception in CheckExternalStates [TypeError: Cannot read property 'val' of null]
2021-08-20 09:29:27.548 - info: heatingcontrol.0 (22372) checking external states done

</body>
</html>

`

**Comment by rg-engineering** [comment]
@looxer01 : ja, habe ich übersehen. Installiere mal bitte von hier und prüfe nochmal. Wenn es dann passt, schiebe ich es noch nach NPM

**Comment by looxer01** [comment]
vielen Dank, sieht jetzt gut aus
